setup.py

```diff
@@ -325,7 +325,7 @@ def run(self):
     extra_link_args.append('-Wl,-rpath,' + cuda_lib_path)
     extra_compile_args += ['-DWITH_CUDA']
     extra_compile_args += ['-DCUDA_LIB_PATH=' + cuda_lib_path]
-    main_libraries += ['cudart']
+    main_libraries += ['cudart', 'nvToolsExt']
```

Comment:
I think this is the right way to do it to ensure that you load the correct nvToolsExt library